website/code_issues.qmd

```diff
@@ -80,8 +80,55 @@ In your examples, vignettes, demos and tests setting a seed is not only allowed
 
 -----------------------------------------------------------
 
-# Issue 3
+# Using print()/cat()
 
 ## Problem
 
+You are using functions, like print() or cat(), to print unsuppressable output to the console.
+
 ## Solution
+
+Change print()/cat() to message(), warning(), stop(), or wrap them in if(verbose){}.
+
+### Details
+Information messages, like loop counts or status updates, can clutter the console. While some users prefer this display, others appreciate less information on their console. The use of printing functions for console output is not forbidden on CRAN. However, this output must be suppressable by users.
```

Comment:
I would comment on when to use `print()` or `cat()` functions. Like using `cat()` for print methods of classes. (I am not actually sure when I would use `print()` in a package.)

Reply:
It would be useful to put a code example of how to implement the verbose usage or maybe link to an existing example like [here](https://www.educative.io/answers/what-is-verbose-in-r)
```
if(verbose){

}
```

Reply:
scratch this comment - i see the example you added below 😄